somaticseq/somatic_vcf2tsv.py

```diff
@@ -511,25 +515,26 @@ def vcf2tsv(is_vcf=None, is_bed=None, is_pos=None, nbam_fn=None, tbam_fn=None, t
                     
                     #################### Collect SomaticSniper ####################:
                     if sniper:
+                        sniper_classification, score_somaticsniper = annotate_caller.SomaticSniper(variant_id, sniper_variants)
                         
-                        if variant_id in sniper_variants:
+                        # if variant_id in sniper_variants:
```

Comment:
Thanks for refactoring. You should remove the commented out code once the refactoring is done.